ee/control/actionqueue/actionqueue.go

```diff
@@ -128,6 +130,7 @@ func (aq *ActionQueue) Update(data io.Reader) error {
 				"failed to do action with action, not marking action complete",
 				"err", err,
 			)
+			processError = fmt.Errorf("actor.Do failed: %w", err)
```

Comment:
Maybe we should return the action type in the error and add to logs above? Might help with debugging in the future.

```suggestion
			processError = fmt.Errorf("actor.Do, action type: %s, failed: %w", action.type, err)
```